scripts/gallery/cache/numpy_team.context

```diff
@@ -0,0 +1,1059 @@
+ignore_io_errors: false
```

Comment:
Is this file supposed to be committed? The fact it dupicates the yaml makes me think that perhaps it shouldn't be.

Reply:
Yes, see the discussion above - it's the output of retrieving from the GitHub API, which we don't want to do all the time.